### PR TITLE
New rule `selector-type-no-unknown`.

### PR DESCRIPTION
**Pull request by evilebottnawi**
/cc @davidtheclark

### CODE REVIEW

File: src/rules/selector-type-no-unknown/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+```css
+input { }
+```
+
+```css
+iNpUt { }
+```
+
+```css
+INPUT { }
```
Comment (davidtheclark) [inline src/rules/selector-type-no-unknown/README.md]: In this and other docs, I think we probably don't want to devote so much space to demonstrate case insensitivity. We could just add a sentence that it's case insensitive. But with this much space devoted to it, it seems like a major feature of the rule, one of the most important, when it's really not.

**Comment by davidtheclark** [comment]
LGTM, besides the one comment about case sensitivity in the docs.

File: src/rules/selector-type-no-unknown/README.md (hunk 1 above)
Comment (evilebottnawi) [inline src/rules/selector-type-no-unknown/README.md]: @davidtheclark agree, which doc need edit to this?

**Comment by evilebottnawi** [comment]
@davidtheclark i think after i add all `SVG` tags, we should merge this PR and next PR i remove all case insensitivity example and add common remark in docs about this.

**Comment by davidtheclark** [comment]
OK

**Pull request merged**